ToNote/Controls/ExtendedRichTextBox.cs

```diff
@@ -83,6 +85,38 @@ public ExtendedRichTextBox()
                     }
                 }
             };
+
+            this.AllowDrop = true;
+
+            this.PreviewMouseMove += (s, e) =>
+            {
+                var point = e.GetPosition(this);
+
+                if (point.X <= 5)
```

Comment:
Ternary would be nicer, `Mouse.OverrideCursor = point.X <= 5 ? Cursors.SizeAll : null;`